vnext/Microsoft.ReactNative/Views/TextInputViewManager.cpp

```diff
@@ -804,6 +806,18 @@ ShadowNode *TextInputViewManager::createShadow() const {
 
 XamlView TextInputViewManager::CreateViewCore(int64_t /*tag*/, const winrt::Microsoft::ReactNative::JSValueObject &) {
   xaml::Controls::TextBox textBox;
+  // This works around a XAML Islands bug where the XamlRoot of the first
+  // window the flyout is shown on takes ownership of the flyout and attempts
+  // to show the flyout on other windows cause the first window to get focus.
+  // https://github.com/microsoft/microsoft-ui-xaml/issues/5341
+  if (IsXamlIsland() &&
+      winrt::Windows::Foundation::Metadata::ApiInformation::IsApiContractPresent(
+          L"Windows.Foundation.UniversalApiContract", 7)) {
+    xaml::Controls::TextCommandBarFlyout flyout;
+    flyout.Placement(xaml::Controls::Primitives::FlyoutPlacementMode::BottomEdgeAlignedLeft);
```

Comment:
should this be BottomEdgeAlignedRight for RTL?

Reply:
Good question... let me check.

Reply:
I set FlowDirection to RightToLeft and the Placement was still BottomEdgeAlignedLeft in a standalone UWP app. Anything else I should test?

Reply:
I also set my system settings to Hebrew and the default was still BottomEdgeAlignedLeft. That being said, it looked like the app deployed in RTL mode (window Minimize/Maximize/Close buttons were on the top left), then when XAML rendered it switched back to LTR.

Reply:
thanks for checking : ) looks fine then!